PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/ServiceBusReceiverAsyncClientCheck.java

```diff
@@ -25,7 +25,6 @@ public class ServiceBusReceiverAsyncClientCheck extends LocalInspectionTool {
     private static List<String> CLIENT_DATA = null;
 
     // Define constants for string literals
-    private static final String RULE_CONFIGURATION = "META-INF/ruleConfigs.json";
     private static final String RULE_NAME = "ServiceBusReceiverAsyncClientCheck";
     private static final String SUGGESTION_KEY = "antipattern_message";
```

Comment:
Move these constants to utils and share them between rules.

Reply:
resolved in Json config PR